src/columnresizing.ts

```diff
@@ -50,6 +57,7 @@ export type Dragging = { startX: number; startWidth: number };
 export function columnResizing({
   handleWidth = 5,
   cellMinWidth = 25,
+  defaultCellMinWidth = 100,
```

Comment:
While I agree with the intention behind the PR, I just spent a bunch of time trying to figure out where this 100px min-width was coming from since it was breaking my tests at https://github.com/ueberdosis/tiptap

I think this might actually be considered a breaking change, because now the defaults have changed from being a minimum of 25px to a minimum of 100px. So rendering the same table with the version prior & the new version _can render differently_

The only path that I see forward here is to change the default of `100` to be `25` or to `cellMinWidth` to keep the previous behavior but maintain the new capability (and later on we can introduce change in a major later)

@ocavue  @YousefED